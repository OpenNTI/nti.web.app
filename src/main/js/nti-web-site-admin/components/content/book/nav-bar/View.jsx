import React from 'react';

import Card from '../../../common/Card';

import Identity from './Identity';
import Tabs from './Tabs';

export default function SiteAdminBookNavBar (props) {
	return (
		<Card className="site-admin-course-nav-bar">
			<Identity {...props} />
			<Tabs {...props} />
		</Card>
	);
}
