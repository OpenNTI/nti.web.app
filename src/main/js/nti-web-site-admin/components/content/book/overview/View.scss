@import '~nti-style-common';

.site-admin-book-overview {
	@include clearfix;

	.site-admin-active-days {
		width: 100%;
		margin-top: 5px;
	}

	.active-users {
		width: 100%;

		.dashboard-list-widget {
			margin: 0 0 10px;
		}
	}

	.active-times-widget {
		margin: 0 0 10px;
	}

	.site-admin-active-days {
		margin: 10px 0;
	}

	.site-admin-row {
		display: flex;

		.site-admin-card {
			width: 250px;
			height: 320px;

			.value {
				padding-top: 1rem;
			}
		}

		.active-users {
			height: 320px;

			.header {
				margin-top: 0;
				margin-bottom: 0;
			}
		}
	}
}
