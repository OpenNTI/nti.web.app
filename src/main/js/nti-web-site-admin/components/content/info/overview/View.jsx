import PropTypes from 'prop-types';

import { AdminTools } from '@nti/web-course';

SiteAdminCourseOverview.propTypes = {
	course: PropTypes.object,
};
export default function SiteAdminCourseOverview({ course }) {
	return <AdminTools.Dashboard course={course} />;
}
