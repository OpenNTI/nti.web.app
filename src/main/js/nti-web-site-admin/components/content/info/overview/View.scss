@import '~nti-style-common';

.site-admin-course-overview {
	@include clearfix;

	.site-admin-active-days {
		width: 100%;
		margin-top: 5px;
	}

	.site-admin-row {
		display: flex;

		.site-admin-card {
			width: 250px;
			height: 320px;

			.value {
				padding-top: 1rem;
			}
		}

		.active-users {
			margin: 0 0 0 4px;
			height: 320px;

			.header {
				margin-top: 0;
				margin-bottom: 0;
			}
		}
	}
}
