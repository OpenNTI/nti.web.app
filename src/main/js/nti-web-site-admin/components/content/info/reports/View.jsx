import React from 'react';
import PropTypes from 'prop-types';

import { List } from '@nti/web-reports';

import Card from '../../../common/Card';

SiteAdminCourseReports.propTypes = {
	course: PropTypes.object,
};
export default function SiteAdminCourseReports({ course }) {
	return course ? (
		<Card>
			<List context={course} />
		</Card>
	) : null;
}
