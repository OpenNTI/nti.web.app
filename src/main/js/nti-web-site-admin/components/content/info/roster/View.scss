@import '~@nti/style-common';

.nti-site-admin-course-roster {
	.site-admin-course-roster-header {
		display: flex;
		flex-direction: row;
		justify-content: flex-end;
		margin-bottom: 1rem;
	}
}

.site-admin-course-roster {
	.loading-mask {
		position: relative;
		min-height: 200px;
	}

	ul {
		li {
			box-shadow: 0 1px 0 0 var(--border-grey)-light;

			a {
				text-decoration: none;
				display: block;
			}
		}
	}

	.controls {
		span {
			color: var(--primary-blue);
			margin-right: 10px;
			cursor: pointer;
		}
	}
}
