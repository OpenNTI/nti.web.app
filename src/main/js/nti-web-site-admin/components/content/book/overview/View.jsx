import './View.scss';
import React from 'react';
import PropTypes from 'prop-types';
import { Widgets } from '@nti/web-reports';

const { ActiveDays, ActiveUsers, ActiveTimes } = Widgets;

export default class SiteAdminBookOverview extends React.Component {
	static propTypes = {
		course: PropTypes.object
	}

	state = {}

	componentDidMount () {
		const {course} = this.props;

		if(!this.state.resp && course && course.hasLink('users')) {
			course.fetchLink('users').then(resp => {
				this.setState({resp});
			});
		}
	}

	render () {
		const {course} = this.props;

		return (
			<div className="site-admin-book-overview">
				<div className="site-admin-row">
					<div className="active-users">
						<ActiveUsers entity={course}/>
					</div>
				</div>
				<div className="active-days">
					<ActiveDays entity={course}/>
				</div>
				<div className="active-times">
					<ActiveTimes course={course}/>
				</div>
			</div>
		);
	}
}
