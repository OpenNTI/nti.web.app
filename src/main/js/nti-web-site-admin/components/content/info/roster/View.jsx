import './View.scss';
import React from 'react';
import PropTypes from 'prop-types';
import {Roster, Enrollment} from '@nti/web-course';
import {List, Loading, Button} from '@nti/web-commons';
import {scoped} from '@nti/lib-locale';
import {LinkTo} from '@nti/web-routing';

import ErrorMessage from '../../../common/ErrorMessage';
import Card from '../../../common/Card';

import Item from './Item';

const DEFAULT_TEXT = {
	error: 'Unable to load roster.',
	manage: 'Manage Roster'
};
const t = scoped('nti-site-admin.courses.course.info.Roster', DEFAULT_TEXT);

export default class SiteAdminCourseRoster extends React.Component {
	static propTypes = {
		course: PropTypes.object
	}


	attachRoster = x => this.roster = x;

	onChange = () => {
		if (this.roster) {
			this.roster.reload();
		}
	}


	render () {
		const {course} = this.props;

		return (
			<div className="nti-site-admin-course-roster">
				{this.renderHeader(course)}
				<Roster ref={this.attachRoster} course={course} renderRoster={this.renderRoster} />
			</div>
		);
	}


	renderRoster = ({loading, error, items, loadNextPage, loadPrevPage}) => {
		return (
			<div className="site-admin-course-roster">
				{loading && (<div className="loading-mask"><Loading.Mask /></div>)}
				{error && (<ErrorMessage>{t('error')}</ErrorMessage>)}
				{!loading && this.renderItems(items)}
				{!loading && this.renderControls(loadPrevPage, loadNextPage)}
			</div>
		);
	}


	renderHeader (course) {
		const {CatalogEntry} = course;

		return (
			<div className="site-admin-course-roster-header">
				<Enrollment.Admin.Prompt.Trigger course={CatalogEntry} onChange={this.onChange}>
					<Button rounded>
						{t('manage')}
					</Button>
				</Enrollment.Admin.Prompt.Trigger>
			</div>
		);
	}


	renderItems (items) {
		return (
			<Card>
				<List.Unadorned>
					{items.map((item, index) => {
						return (
							<li key={index}>
								<LinkTo.Object object={item} context="site-admin.courses.course-roster.list">
									<Item item={item} onChange={this.onChange} />
								</LinkTo.Object>
							</li>
						);
					})}
				</List.Unadorned>
			</Card>
		);
	}


	renderControls (loadPrevPage, loadNextPage) {
		return (
			<div className="controls">
				{loadPrevPage && (<span onClick={loadPrevPage}>Prev</span>)}
				{loadNextPage && (<span onClick={loadNextPage}>Next</span>)}
			</div>
		);
	}
}
