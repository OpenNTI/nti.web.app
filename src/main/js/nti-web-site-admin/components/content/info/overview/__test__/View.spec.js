import React from 'react';
import renderer, { act } from 'react-test-renderer';

import * as TestUtils from '@nti/web-client/test-utils';
import { flushPromises } from '@nti/lib-commons/test-utils';

import View from '../View';

const { tearDownTestClient, setupTestClient } = TestUtils;

const getMockService = () => {
	return {
		getObject: () => {
			return {
				title: 'abc',
			};
		},
		get: link => {
			if (link === 'mockAnalytics') {
				return Promise.resolve({
					Links: [
						{
							rel: 'active_times_summary',
							href: 'mockActiveTimes',
						},
					],
				});
			} else if (link === 'mockActiveTimes') {
				return Promise.resolve({});
			}
		},
	};
};

const onBefore = () => {
	jest.useFakeTimers();
	setupTestClient(getMockService());
};

const onAfter = () => {
	tearDownTestClient();
};

/* eslint-env jest */
describe('Site admin user course overview test', () => {
	beforeEach(() => onBefore());
	afterEach(onAfter);

	test('Basic render test', async () => {
		const course = {
			hasLink: () => true,
			fetchLink: () => {
				return Promise.resolve({
					Items: [],
				});
			},
			Links: [
				{
					rel: 'analytics',
					href: 'mockAnalytics',
				},
			],
			getLink: () => 'mockEntityLink',
		};

		let cmp;
		act(() => {
			cmp = renderer.create(<View course={course} />);
		});

		jest.runAllTimers();
		await flushPromises();
		jest.runAllTimers();

		const tree = cmp.toJSON();

		expect(tree).toMatchSnapshot();
		cmp.unmount();
	});
});
