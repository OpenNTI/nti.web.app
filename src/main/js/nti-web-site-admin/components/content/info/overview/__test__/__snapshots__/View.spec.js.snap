// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Site admin user course overview test Basic render test 1`] = `
<div
  className="course-admin-course-overview"
>
  <div
    className="admin-row"
  >
    <div>
      <div
        className="dashboard-list-widget statistics"
      >
        <div
          className="statistics-content"
        >
          <div
            className="progress"
          >
            <div
              className="circular-progress no-progress"
            >
              <svg
                height={100}
                viewBox="0 0 36 36"
                width={100}
              >
                <path
                  d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831"
                  fill="none"
                  stroke="#eee"
                  strokeDasharray="100, 100"
                  strokeWidth="2"
                />
                <path
                  d="M18 2.0845 a 15.9155 15.9155 0 0 1 0 31.831 a 15.9155 15.9155 0 0 1 0 -31.831"
                  fill="none"
                  stroke="#3fb34f"
                  strokeDasharray="0, 100"
                  strokeWidth="2"
                />
              </svg>
              <div
                className="value"
                style={
                  Object {
                    "fontSize": 33.333333333333336,
                    "top": 25,
                    "width": 100,
                  }
                }
              >
                <span
                  className="number"
                >
                  0
                </span>
                <span
                  className="percent"
                  style={
                    Object {
                      "fontSize": 16.666666666666668,
                    }
                  }
                >
                  %
                </span>
              </div>
            </div>
          </div>
          <div
            className="labeled-value"
          >
            <div
              className="label"
            >
              Total Enrolled
            </div>
            <div
              className="value"
            >
              0
            </div>
          </div>
          <div
            className="labeled-value"
          >
            <div
              className="label"
            >
              Students Completed
            </div>
            <div
              className="value"
            />
          </div>
          <div
            className="labeled-value"
          >
            <div
              className="label"
            >
              Students With Progress
            </div>
            <div
              className="value"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="active-users"
    >
      <div
        className="dashboard-list-widget active-users"
      >
        <div
          className="header"
        >
          <div
            className="title"
          >
            Top Learners
          </div>
          <div
            className="pager"
          >
            <div
              className="page-control previous disabled"
              onClick={[Function]}
            >
              <i
                className="icon-chevron-left"
              />
            </div>
            <div
              className="page-control next disabled"
              onClick={[Function]}
            >
              <i
                className="icon-chevron-right"
              />
            </div>
          </div>
        </div>
        <div
          className="no-items"
        >
          No top learners found
        </div>
      </div>
    </div>
  </div>
  <div
    className="active-days"
  >
    <div
      className="admin-active-days"
    >
      <div
        className="daily-activity-header"
      >
        Daily Activity
      </div>
      <div
        className="no-data"
      >
        No activity found
      </div>
    </div>
  </div>
  <div
    className="active-times"
  >
    <div
      className="active-times-widget"
    >
      <div
        className="active-times-header"
      >
        Active Times
      </div>
      <div
        className="active-times-chart"
      >
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            12 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            2 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            4 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            6 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            8 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            10 am
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            12 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            2 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            4 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            6 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            8 pm
          </div>
        </div>
        <div
          className="active-blocks"
        >
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="active-block"
            style={
              Object {
                "backgroundColor": "#efefef",
              }
            }
          />
          <div
            className="time-label"
          >
            10 pm
          </div>
        </div>
        <div
          className="days-of-week"
        >
          <div
            className="day"
          >
            Sun
          </div>
          <div
            className="day"
          >
            Mon
          </div>
          <div
            className="day"
          >
            Tues
          </div>
          <div
            className="day"
          >
            Wed
          </div>
          <div
            className="day"
          >
            Thur
          </div>
          <div
            className="day"
          >
            Fri
          </div>
          <div
            className="day"
          >
            Sat
          </div>
        </div>
      </div>
      <div
        className="active-times-popular"
      >
        <div
          className="active-times-popular-header"
        >
          Most Popular Time
        </div>
        <div
          className="active-times-popular-content"
        >
          No Activity Found
        </div>
      </div>
    </div>
  </div>
</div>
`;
